build/config_dev.go

```diff
@@ -0,0 +1,137 @@
+//go:build dev
+// +build dev
+
+package build
+
+import (
+	"fmt"
+	"strings"
+
+	btclogv1 "github.com/btcsuite/btclog"
+	"github.com/btcsuite/btclog/v2"
+)
+
+const (
+	resetSeq = "0"
```

Comment:
Could you explain why we are only adding style option to the dev env, because of some attack vectors mentioned in the btclog PR ?

Reply:
we can always change this later I think. Just feels good to be safe for now in this initial PR